### PR TITLE
Use string.GetHashCode for NET6

### PR DESCRIPTION
**Pull request by sungam3r**

### CODE REVIEW

**Comment by Shane32** [comment]
You changed the ROM implementation, but your demo test does not use the ROM.GetHashCode() method.  It is using `ReadOnlyMemory<char>.GetHashCode`.  The test passes just fine with this change to the test:

```cs
        var h1 = new ROM(s1).GetHashCode();
        var h2 = new ROM(s2).GetHashCode();
```

**Comment by Shane32** [comment]
The test also passes like this:
```cs
        var h1 = string.GetHashCode(s1.Span);
        var h2 = string.GetHashCode(s2.Span);
```

File: src/GraphQLParser/GraphQLParser.csproj [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <Description>Library containing lexer and parser for GraphQL syntax</Description>
     <Product>GraphQL Parser</Product>
     <AssemblyTitle>GraphQL Parser for .NET</AssemblyTitle>
-    <TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;net6</TargetFrameworks>
```
Comment (sungam3r) [inline src/GraphQLParser/GraphQLParser.csproj:7]: The package will increase by a third.

**Comment by sungam3r** [comment]
Yes my mistake.

**Review by Shane32 — APPROVED** [review]

**Pull request merged**